webside/auth.py

```diff
@@ -72,13 +72,9 @@ def sign_up():
 
 
 @auth.route('/shared')
+@login_required
 def shared():
-    user = current_user
-    shared_notes = user.shared_notes.all()
-    shared_by_users = {note.id: note.user.username for note in shared_notes}
     session.pop('new_items', None)
+    current_user.reset_last_checked()
```

Comment:
Nie jest to konieczność ale często w przypadku takich framework'ów tworzy się albo funkcje zewnętrzne ale methody na modelu który wynuje jakies zmiany na instancji.

W tym przypadku albo:
```python
reset_last_checked(current_user)
```

albo 
```python

class User:
    ....
    
    def reset_last_checked(self):
         pass
    
current_user.reset_last_checked()
```

Troche zalezy od struktury jaka się przyjmie

Reply:
W tym widoku jeszcze uprościłem duzo bo wiekszosc rzeczy juz była dostępna